### ISSUE TITLE
Select inputs for month and year should have hidden label or aria-label attribute

### ISSUE THREAD

**Issue by heutonaj** [onefinestay/react-daterange-picker#219]
Consider adding a hidden label or an aria-label attribute to the select inputs of month and year on the calendar.
